lib/gitsh/environment.rb

```diff
@@ -88,7 +87,7 @@ def repo_has_untracked_files?
     end
 
     def repo_config_color(name, default)
-      if color_override = self[name]
+      if color_override = fetch(name) { nil }
```

Comment:
If the key doesn't exist, `#fetch` will yield to the block for the default value. In this case, the default value is `nil`.